openghg/analyse/_scenario.py

```diff
@@ -249,19 +249,19 @@ def _get_data(self, keywords: ParamType, data_type: str) -> Any:
             except (SearchError, AttributeError):
                 num = i + 1
                 if num == num_checks:
-                    print(f"Unable to add {data_type} data based on keywords supplied.")
-                    print(" Inputs -")
+                    logger.info(f"Unable to add {data_type} data based on keywords supplied.")
```

Comment:
Should we elevate these to warnings so they get seen directly by the user when being run? (I believe that's how we've set that up..)

Reply:
Done